runner/runner.go

```diff
@@ -15,6 +15,8 @@ import (
 	"sync/atomic"
 	"syscall"
 	"time"
+	
+	"github.com/yookoala/gofast"
```

Comment:
can we use `github.com/kanocz/fcgi_client` here, so it shares dependencies with https://github.com/Automattic/php-fpm-exporter

Reply:
That was going to be my initial approach, but it doesn't look like that project is still active. There was another reason I decided to go with this other fgci client implementation, but I'll have to take another look at fcgi_client to remember what that reason was.